src/main/java/com/endava/internship/collections/Tests.java

```diff
@@ -0,0 +1,31 @@
+package com.endava.internship.collections;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.*;
+
+import java.time.LocalDate;
+
+public class Tests {
+    StudentSet students = new StudentSet();
```

Comment:
you can initialise a new StudentSet in a constructor and use @BeforeEach to make sure that set is empty before tests are run

Reply:
done